File: CPP/Assignment/day7/seperateStudent/Student.cpp
 
#include "Student.h"
 using namespace std;
 
    void Student::enterDetails()
    {
            cout << "enter student rollNo:->" << endl;
            cin >> rollNo;
            cout << "Enter Name: ";
            cin >>name;
            cout << "Enter Age: ";
            cin >>age;
            cout << "Enter Mobile Number: ";
            cin >>phoneNumber;

    }

    void Student::displayDetails()
    {
            cout << "\nDetails of Students:" << endl;
            cout<<"Roll No. : " <<rollNo<<endl;
            cout<<"Name : "<<name<<endl;
            cout<<"Age : "<<age<<endl;
            cout<<"Phone Number : "<<phoneNumber<<endl;

    }

    int Student::getRollNo() const{
        return rollNo;
    }
    string Student::getName() const{
        return name;
    }

    bool Student::compare(const Student &s1, const Student &s2)
    {
        if (s1.getRollNo() != s2.getRollNo())
            return s1.getRollNo() < s2.getRollNo();
        return s1.getName() < s2.getName();
    }
 
 

File: AlgoAndDSA/Assignment/Day3/Queue/src/com/acts/Student.java
package com.acts;

public class Student {
	private int roll;
	private String name;
	private int age;
	private static int genRoll = 100;
	
	public Student(String name, int age) {
		super();
		this.roll = genRoll++;
		this.name = name;
		this.age = age;
	}

	@Override
	public String toString() {
		return "Student [roll=" + roll + ", name=" + name + ", age=" + age + "]";
	}
	
	
}
